Bottom: a265e2972773df3fedad123f406805f194db2805
Top:    0f7bbdf2b267ed0985140edbd7dc277d9717751e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-01 11:20:23 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 384440e..73dfc14 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2267,7 +2267,7 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
   glpixmap = gdk_pixmap_set_gl_capability (pixmap, glconfig, NULL);
   gldrawable = GDK_GL_DRAWABLE (glpixmap);
   glcontext = gdk_gl_context_new (gldrawable, drawarea_glcontext,
-                                  FALSE, GDK_GL_RGBA_TYPE);
+                                  TRUE, GDK_GL_RGBA_TYPE);
 
   /* Setup zoom factor for drawing routines */
